build/common/installer/scripts/tomlparser.rb

```diff
@@ -79,6 +86,15 @@ def populateSettingValuesFromConfigMap(parsedConfig)
             end
           end
         end
+
+        if @collectStdoutLogs && !stdoutSystemPods.nil? && stdoutSystemPods.kind_of?(Array)
+          # Checking only for the first element to be string because toml enforces the arrays to contain elements of same type
+          if stdoutSystemPods.length > 0 && stdoutSystemPods[0].kind_of?(String)
+            @stdoutIncludeSystemPods = stdoutSystemPods.join(",")
```

Comment:
If customer uses the namespaces other than system namespaces list, we should log warn message and ignore the entry.